rector.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Rector configuration
+ *
+ * @package Parsely
+ */
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Php54\Rector\Array_\LongArrayToShortArrayRector;
+use Rector\Set\ValueObject\LevelSetList;
+
+return static function ( RectorConfig $rector_config ): void {
+	$rector_config->paths(
+		array(
+			__DIR__ . '/src',
+			__DIR__ . '/tests',
+			__DIR__ . '/views',
+			__DIR__ . '/uninstall.php',
+			__DIR__ . '/wp-parsely.php',
+		)
+	);
+
+	// Define sets of rules.
```

Comment:
Comment seems extra. By the way how about if we change the filename to `rector.config.php` .... think it conveys more meaning 🤔

Reply:
The default output from `vendor/bin/rector init` is to create a `rector.php`. Looking at https://getrector.org/documentation/config-configuration, then you can indeed name it what you want, but then you'll need to pass a `--config` option with the new name for every `rector` command. That seems like something that would be forgotten and get frustrating than using the default config file name.

> Comment seems extra.

For someone not overly familiar with Rector or its config file, I'll take whatever guidance I can for now :-)